### PR TITLE
Create CLI for Library

### PR DESCRIPTION
**Pull request by petushka1**
# Milestone 4
### Requirements
- [x] ensure there are [no linter errors](https://github.com/microverseinc/linters-config).
- [x] follow [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- [x] documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

- [x] Create app.rb and build functionality to:

- List all books.
- List all people.
- Create a person (teacher or student, not a plain Person).
- Create a book.
- Create a rental.
- List all rentals for a given person id.

- [x] create main.rb and build the following functionality:

- Present the user with a list of options to perform.
- Lets users choose an option.
- If needed, ask for parameters for the option.
- Have a way to quit the app.

### CODE REVIEW

File: src/main.rb [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+def create_student(app)
+  puts 'Enter Student [Name]'
+  name = gets.chomp
+  puts 'Enter Student [Age]'
+  age = gets.chomp
+  app.create_student(age, name)
+  puts "New Student added:\n[Student] ID: #{app.students.last.id}, "
+  print "Name: #{app.students.last.name}, Age: #{app.students.last.age}"
+end
```
Comment (mikethreels) [inline src/main.rb]: - [ ] for a student the app is supposed to check if they have parent permission.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Nadia, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
- professional looking Readme file

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/main.rb (hunk 1 above)
Comment (petushka1) [reply src/main.rb]: # Improvements
- check if age < 18
- require setting permission
- if age >= 18 sets permission to true automatically

Thank you for your advice and assistance.

Comment (Whoistolu) [reply src/main.rb]: Well resolved 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @petushka1 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**